src/library/library.cpp

```diff
@@ -172,6 +172,9 @@ Library::Library(
     } else {
         m_trackTableFont = QApplication::font();
     }
+
+    m_editMetadata = m_pConfig->getValue(
+            ConfigKey(kConfigGroup, "EditMetadata"), true);
```

Comment:
The names in the code should have more context as well as the string shown to the user in the preferences. How about `m_editMetadataSelectedClick` for the boolean and `EditMetadataSelectedClick` for the ConfigKey name?